### ISSUE TITLE
Incorrect Job is not going back to be reqo

### ISSUE THREAD

**Issue by rmdedios** [wrktech/community#19]
https://community.wrk.com/wrkstation/50ef2ddc-31c8-4004-b6f1-8901b985a5dd

In the job the info required from the previous job is blank
![Missing regulation](https://user-images.githubusercontent.com/61804162/82577267-b8518900-9b58-11ea-9b7e-19085d2ef80c.PNG)

When checked the tracing tools, Job ID: 24de0b6a-d9a5-4d72-936f-7ac5a2236320 was invalid
![regmissing2](https://user-images.githubusercontent.com/61804162/82577289-be476a00-9b58-11ea-9ce5-5f2e71f39364.PNG)

Job: Data Entry
Job ID: 24de0b6a-d9a5-4d72-936f-7ac5a2236320 ### **(Invalid)**
Completion Status: Complete Job
Completed by: , (sodevadas)
Generated data:
765dcb66-f35d-436c-9b1d-d6054a3fe32f
Job: Data Entry (Supervision Job)
Job ID: cc722432-af3a-476c-82f6-e9cf65b2bf8c
Completion Status: **The Job is Incorrect**
Completed by: , (timfwadsworth)
Generated data:

This job when to the next job which is classify, when it should have gone back to data entry to be worked on again.

**Comment by robbrit** [comment]
I don't really understand the problem. Can you rephrase?

**Comment by rmdedios** [comment]
@robbrit 

Job is completed by the user and goes to supervision
Superior says Job is Incorrect - the job is supposed to go back to redone but instead it's going to the next job with empty details. 
![issuessss](https://user-images.githubusercontent.com/61804162/82828622-7ccbfd00-9e7f-11ea-81ac-596e821fc4a5.PNG)


another example: https://community.wrk.com/wrkstation/426bd6f3-5ea2-4147-b12b-9d91214ca336
